typescript/agentkit/src/action-providers/onramp/utils/getOnrampBuyUrl.ts

```diff
@@ -0,0 +1,38 @@
+import type { GetOnrampUrlWithProjectIdParams, GetOnrampUrlWithSessionTokenParams } from "./types";
+import { ONRAMP_BUY_URL, VERSION } from "./verion";
+
+/**
+ * Builds a Coinbase Onramp buy URL using the provided parameters.
+ *
+ * @param props - Configuration options for the Onramp buy URL
+ * @param props.projectId - A projectId generated in the Coinbase Developer Portal
+ * @returns The generated Onramp buy URL
+ */
+export function getOnrampBuyUrl({
```

Comment:
I decided to manually grab the code rather than import it because the OnchainKit library this exists in is a frontend library. It would be adding irrelevant dependencies when all we need is to build a URL, but build it properly